README.md

```diff
@@ -268,6 +269,7 @@ Habitica is a gamified task manager, webapp and android/ios app, really wonderfu
 - [MindsDB](https://github.com/mindsdb/mindsdb/labels/good%20first%20issue) _(label: good first issue)_ <br> MindsDB is an open source AI layer for existing databases.
 - [Bokeh](https://github.com/bokeh/bokeh/labels/good%20first%20issue) _(label: good first issue)_ <br> Bokeh is an interactive visualization library for modern web browsers.
 - [SymPy](https://github.com/sympy/sympy/labels/Easy%20to%20Fix) _(label: Easy to Fix)_ <br> A Python library for symbolic mathematics.
+- [STVIR/pysot](https://github.com/STVIR/pysot/labels/good-first-issue)_(label: good-first-issue)_ <br>SenseTime Research Platform for single object tracking,implementing algorithms like SiamRPN and SiamMask.
```

Comment:
Please update link and label in this one - it has no `-` in there

https://github.com/STVIR/pysot/labels/good%20first%20issue